atramhasis/views/exception_views.py

```diff
@@ -64,7 +64,7 @@ def provider_unavailable(exc, request):
     """
     log.error(exc, exc_info=sys.exc_info())
     request.response.status_int = 503
-    return {'message': exc.message}
+    return {'message': exc}
```

Comment:
Deze was goed. De ProviderUnavailableException heeft een `message`, die niet perse hetzelfde is als de string van de exception.